testutil/sample/crypto.go

```diff
@@ -91,15 +92,23 @@ func EthAddressFromRand(r *rand.Rand) ethcommon.Address {
 }
 
 // BtcAddressP2WPKH returns a sample btc P2WPKH address
-func BtcAddressP2WPKH(t *testing.T, net *chaincfg.Params) string {
+func BtcAddressP2WPKH(t *testing.T, net *chaincfg.Params) *btcutil.AddressWitnessPubKeyHash {
```

Comment:
Is it possible to refactor this function to accept randomness as an input? 
Can be used to generate the private key  on line 96